src/main/python/janusgraph_python/driver/ClientBuilder.py

```
@@ -0,0 +1,89 @@
+
+        Args:
+            host (str): The HOST of JanusGraph gremlin-server instance. Defaults to localhost
+            port (str): The PORT of JanusGraph gremlin-server instance. Defaults to 8182.
+            graph (str): The GraphTraversalSource being exposed from gremlin-server instance. Defaults to g
+
+        Keyword Args:
+            graphson_reader (GraphSONReader): GraphSONReader object with required Deserializers registered
+            graphson_writer (GraphSONWriter): GraphSONWriter object with required Serializers registered
+
+        Raises:
+            AttributeError: When invalid Keyword arguments is provided. The expected key needs to be
+            `graphson_reader` and `graphson_writer`.
+
+        Returns:
+            JanusGraphClient
+        """
+
+        URL = "ws://{}:{}/gremlin".format(host, port)
+
+        if not kwargs:
+
+            graphson_reader = JanusGraphSONReader().build()
+            graphson_writer = JanusGraphSONWriter().build()
+
+        else:
+            if "graphson_reader" in kwargs and "graphson_writer" in kwargs:
+                graphson_reader = kwargs["graphson_reader"]
+                graphson_writer = kwargs["graphson_writer"]
+            else:
+                raise AttributeError("Additional parameters if provided needs to be keywords arguments of "
+                                     "`graphson_reader` and `graphson_writer`")
+
+        self.REMOTE_CONNECTION = DriverRemoteConnection(URL, traversal_source, graphson_reader=graphson_reader,
+                                                        graphson_writer=graphson_writer)
+
+        return self
+
+    def get_connection(self):
+        """ Get the RemoteConnection object, so that same can be used to create GraphTraversalSource.
+
+        Returns:
+            DriverRemoteConnection
+        """
+
+        return self.REMOTE_CONNECTION
+
+    def close(self):
+        try:
```

Is the try/except really necessary? Wouldn't it be better to just let the user deal with the exception instead of swallowing it and just returning `False`?